source/administration/upgrade.rst

```diff
@@ -130,6 +130,10 @@ If they still show yellow, then you need to trigger a config propagation across
 
 After the server is upgraded, users might need to refresh their browsers to experience any new features.
 
+.. note::
+
+  We only support a one version difference between the server versions when performing a rolling upgrade. Running two different versions of Mattermost in your cluster should not be done outside of an upgrade scenario. 

```

Comment:
```suggestion
  We only support a one minor version difference between the server versions when performing a rolling upgrade. Running two different versions of Mattermost in your cluster should not be done outside of an upgrade scenario. 

```

Maybe this needs to have an example too: e.g.
* 5.27.1 + 5.27.2 is good
* 5.26.4 + 5.27.1 is good
* 5.25.5 + 5.27.0 is bad